healthScore/templates/header.html

```diff
@@ -117,10 +117,7 @@
                 
                 {% else %}
                 <ul class="nav col-12 col-lg-auto mb-2 justify-content-center mb-md-0">
-                    <li><a href="#" class="nav-link">Features</a></li>
-                    <li><a href="/hs/requestHealthHistory" class="nav-link">Request</a></li>
-                    <li><a href="#" class="nav-link">FAQs</a></li>
-                    <li><a href="#" class="nav-link">About</a></li>
+                    <li><a href="/hs/requestHealthHistory" class="nav-link">Request Record</a></li>
```

Comment:
Can you please update it to `Request Health History` for better clarity?

Reply:
Alright!

Reply:
Thanks!!